consoles/sshVirtsh.pm

```diff
@@ -352,9 +353,16 @@ sub _copy_image_vmware ($self, $name, $backingfile, $file_basename, $vmware_open
 }
 
 sub _copy_image_else ($self, $file, $file_basename, $basedir) {
-    $self->run_cmd(sprintf("rsync -av '$file' '$basedir/%s'", $file_basename)) && die 'rsync failed';
+    if (-e $file_basename && defined which 'rsync') {    # utilize asset possibly cached by openQA worker
+        my %c = $self->get_ssh_credentials;
+        bmwqemu::diag "Syncing '$file' directly from worker host to $c{hostname}";
+        system "RSYNC_PASSWORD='$c{password}' rsync -av '$file_basename' '$c{username}\@$c{hostname}:$basedir/$file_basename'";
```

Comment:
I don't think a plain system call is a better alternative to the run_cmd call from before

Reply:
`$self->run_cmd()` would execute the command on the wrong machine.

Reply:
> to the run_cmd call from **before**

Note that the `run_cmd` call is also still there as a fallback. It executed the specified command on the svirt host. However, in my newly added branch `rsync` is simply run on the worker host (targeting the svirt host).